project.clj

```diff
@@ -119,6 +119,9 @@
   ;; code that we have.
   :classifiers  [["test" :testutils]]
 
+  ;; Inovke "lein test :all" to run all tests
+  :test-selectors {:default (fn [m] (not (:slow m)))}
```

Comment:
No, I meant, our scripts in ext/ need to now use 'lein test :all' for FOSS. If we merge this in, these tests won't run in CI any more, I'm guessing we don't want that.

https://github.com/puppetlabs/puppetdb/blob/master/ext/jenkins/lein-test.sh